nrtk_explorer/library/nrtk_transforms.py

```diff
@@ -0,0 +1,88 @@
+from typing import Any, Optional, Dict
+
+from PIL.Image import Image
+
+from nrtk_explorer.library.transforms import ImageTransform, ParameterDescription
+
+from nrtk.impls.perturb_image.generic.cv2.blur import GaussianBlurPerturber
+
+
+class NrtkGaussianBlurTransform(ImageTransform):
+    def __init__(self, perturber: Optional[GaussianBlurPerturber] = None):
+        if perturber is None:
+            perturber = GaussianBlurPerturber()
+
+        self._perturber: GaussianBlurPerturber = perturber
+
+    def get_parameters(self) -> Dict[str, Any]:
+        return {
+            "ksize": self._perturber.ksize,
+        }
+
+    def set_parameters(self, params: Dict[str, Any]):
+        self._perturber.ksize = params.get("ksize", 1)
+
+    def get_parameters_description(self) -> Dict[str, ParameterDescription]:
+        ksize_description: ParameterDescription = {
+            "type": "integer",
+            "label": "Kernel Size",
+            "default": 1,
+            "description": None,
+            "options": None,
+        }
+
+        return {
+            "ksize": ksize_description,
+        }
+
+    def execute(self, input: Image) -> Image:
+        return self._perturber.perturb(input)
+
+
+class NrtkPybsmTransform(ImageTransform):
```

Comment:
I remember you mentioned a few weeks ago some nuances with the NRTK pyBSM transform code path- please let me know if you run into any blockers when experimenting with an initial implementation. Thanks!